### PR TITLE
if there are more then 3 items in a camp menu that as the screen size gets smaller menu items disappear #196

### PR DESCRIPTION
**Pull request by tolichka**
Make sure these boxes are checked before asking for review of your pull request - thank you!

## General checks
- [x] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [x] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [x] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [x] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [x] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [x] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [x] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

## Steps for review

- [x] Please provide steps for review here.

1) Go to camps/camp-orkila or any other camp.
on mobile you should be able to drag camp menu to left and right.

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1349 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1349 |
| Installation process | http://install.openy-dev.ffwua.com/build1349/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1349/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1349/profiles/contrib/openy/build/reports/behat

File: themes/openy_themes/openy_rose/scripts/openy_rose.js [hunk 1]
```diff
@@ -106,4 +106,71 @@
     }
   };
 
+  // Horisontal scroll for camp menu.
+  Drupal.behaviors.scrollableList = {
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/scripts/openy_rose.js:115]: Default indentation in Drupal should be 2 spaces. We have 4 here.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

File: themes/openy_themes/openy_rose/scripts/openy_rose.js (hunk 1 above)
Comment (tolichka) [reply themes/openy_themes/openy_rose/scripts/openy_rose.js:115]: Coding styles are done for all file.

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1350 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1350 |
| Installation process | http://install.openy-dev.ffwua.com/build1350/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1350/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1350/profiles/contrib/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1358 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1358 |
| Installation process | http://install.openy-dev.ffwua.com/build1358/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1358/

**Review by alexschedrov — APPROVED** [review]

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1358/profiles/contrib/openy/build/reports/behat

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1359 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1359 |
| Installation process | http://install.openy-dev.ffwua.com/build1359/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1359/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1359/profiles/contrib/openy/build/reports/behat

**Pull request merged**